tests/lib/local.test.ts

```diff
@@ -0,0 +1,55 @@
+import * as fs from 'fs';
+import * as path from 'path';
+import { errors } from '../../src/config/error-config';
+import { createIfNotExists, readFile, deleteFile, createFile } from '../../src/lib/local';
+
+describe('local', () => {
+  describe('createIfNotExists', () => {
+    it('Should create a folder if not already exists', async () => {
+      const folderPath = 'tests/randomFolder';
+
+      createIfNotExists(folderPath);
+
+      expect(fs.existsSync(folderPath)).toBeTruthy();
+      fs.rmdirSync(folderPath);
+    });
+  });
+
+  describe('readFile', () => {
+    it('Should return readStream when file is found', async () => {
+      const result = await readFile(path.resolve(__dirname, '../_helpers/unit_tests.png'));
```

Comment:
I would create a file in `beforeAll` and remove it in `afterAll`. This way we do not need to include files for testing